installer/functions.sh

```diff
@@ -452,10 +452,9 @@ issystemd() {
     # if pid 1 is systemd, it is systemd
     [ "$(basename $(readlink /proc/1/exe) 2>/dev/null)" = "systemd" ] && return 0
 
-    # -- ubuntu runs systemd as init ---
-    ## if systemd is not running, it is not systemd
-    #pids=$(pidof systemd 2>/dev/null)
-    #[ -z "${pids}" ] && return 1
+    # if systemd is not running, it is not systemd
```

Comment:
It would be nice to add a comment that this is here due to a specific setup (ubuntu 14.04)

Reply:
I think it is not.
Our previous assumption was wrong.
Check it again. We don't check if `systemd` is actually running on this system.